src/Dev/SapphireTest.php

```diff
@@ -280,6 +285,50 @@ protected function setUp()
         Email::config()->remove('bcc_all_emails_to');
     }
 
+
+
+    /**
+     * Helper method to determine if the current test should enable a test database
+     *
+     * @param $fixtureFiles
+     * @return bool
+     */
+    protected function setupDatabaseForCurrentTest($fixtureFiles): bool
+    {
+        $databaseEnabledByDefault = $fixtureFiles || $this->usesDatabase;
+
+        return ($databaseEnabledByDefault && !$this->currentTestDisablesDatabase())
+            || $this->currentTestEnablesDatabase();
+    }
+
+    /**
+     * Helper method to check, if the current test uses the database.
+     * This can be switched on with the annotation "@useDatabase"
+     *
+     * @return bool
+     */
+    protected function currentTestEnablesDatabase()
+    {
+        $annotations = $this->getAnnotations();
+
+        return array_key_exists('useDatabase', $annotations['method'])
+            && $annotations['method']['useDatabase'][0] !== 'false';
+    }
+
+    /**
+     * Helper method to check, if the current test uses the database.
+     * This can be switched on with the annotation "@useDatabase false"
+     *
+     * @return bool
+     */
+    protected function currentTestDisablesDatabase()
+    {
+        $annotations = $this->getAnnotations();
+
+        return array_key_exists('useDatabase', $annotations['method'])
```

Comment:
This method is exactly the same as above, with an inverse check. I'd rather a single method that returned either a boolean, or `null` if not declared. These could even be combined with the prior method, as these effectively resolve a single value.

Reply:
Somehow I don't like it to return `null` for "do nothing", as it might be harder to check above in the setup method where this helpers are used to create cleaner code.

is this only used on setup or do we need it also on tear down? We could of course combine those three methods if it's only used for setup (and tear down checks differently if a DB was used)

Reply:
Fair enough.